recipes/missingno/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "missingno" %}
+{% set version = "0.3.5" %}
+{% set sha256 = "385c7bee14b6ce45c86266e5428544ac9e1cad9c3fe55048fe84e5b552308b17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
+    - pandas
+    - matplotlib
+    - seaborn
+
+test:
+  imports:
+    - missingno
+
+about:
+  home: https://github.com/ResidentMario/missingno
+  license: MIT
+  license_family: MIT
+  license_file: {{ environ["RECIPE_DIR"] }}/LICENSE
```

Comment:
My apologies; I was distracted when I made that change. Let me try with the correct way.